Symfony/CS/Fixer/Contrib/OrderedUseFixer.php

```diff
@@ -129,6 +129,18 @@ public function getPriority()
     }
 
     /**
+     * Prepare namespace for sorting.
```

Comment:
what does it mean "prepare" ?

Maybe `clean the namespace from comments and whitespaces` ?